.github/workflows/pr-tests-syft.yml

```diff
@@ -30,6 +30,14 @@ jobs:
 
     runs-on: ${{ matrix.os }}
     steps:
+      - name: "clean .git/config"
+        if: matrix.os == 'windows'
+        continue-on-error: true
+        shell: bash
+        run: |
+          echo "deleting ${GITHUB_WORKSPACE}/.git/config"
```

Comment:
why is this needed?

Reply:
This a permissions on windows self hosted runners, 

Generally during multiple runs, in the docker container tests, the self hosted runners have the concept of new users for each run.

They create temporary files in the pysyft directory.

when it is used in a new run , there are permission issues while actions/checkout

https://github.com/actions/checkout/issues/861